src/Sulu/Bundle/MediaBundle/Resources/public/js/services/format-manager.js

```diff
@@ -0,0 +1,102 @@
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define(['services/husky/util'], function(Util) {
+
+    'use strict';
+
+    var urls = {
+        formats: '/admin/api/media/%id/formats?locale=%locale',
+        format: '/admin/api/media/%id/formats/%key?locale=%locale'
+    };
+
+    return {
+
+        /**
+         * Loads all formats for a given media and a given locale.
+         *
+         * @param {Number} mediaId
+         * @param {String} locale
+         *
+         * @returns {Object} a promise which gets resolved when the formats have been loaded
+         */
+        loadFormats: function(mediaId, locale) {
+            return Util.load(urls.formats.replace('%id', mediaId).replace('%locale', locale));
+        },
+
+        /**
+         * Saves a given format.
+         *
+         * @param {Number} mediaId
+         * @param {String} locale
+         * @param {Object} format
+         * @returns {Object} a promise which gets resolved when the format has been saved
+         */
+        saveFormat: function(mediaId, locale, format) {
+            return Util.save(
+                urls.format
+                    .replace('%id', mediaId)
+                    .replace('%key', format.key)
+                    .replace('%locale', locale),
+                'PUT',
+                format
+            );
+        },
+
+        /**
+         * @param formatWidth The width of the format
+         * @param formatHeight The height of the format
+         * @param imageWidth The width of the image
+         * @param imageHeight The height of the image
+         *
+         * @returns {boolean} True iff an image can be cropped in a format given by its width and height
+         */
+        cropPossibleInInFormat: function(formatWidth, formatHeight, imageWidth, imageHeight) {
+            if (!formatWidth && !formatHeight) {
+                return false;
+            }
+            if ((!!formatWidth && formatWidth > imageWidth) ||
```

Comment:
These ifs could be easily combined into one (if there are reasons not to do so, please insert a new line before this one.